config/vufind/BlenderMappings.yaml

```diff
@@ -0,0 +1,292 @@
+# Fields in each configuration provide mappings from field names used in Blender.ini
+# to field names used in each backend.
+# By default the field names and behavior match those used with Solr.
```

Comment:
I'm not sure exactly what this "By default" statement means. If a missing mapping means a field is unsupported, then in what situation is a default used? Or are you just saying "This example configuration is designed with the assumption that Solr is the primary search backend used"?